### DISCUSSION TITLE
Support different tax rates for different permission groups.

### DISCUSSION THREAD

**Discussion by MSCMDD** [QuickShop-Community/QuickShop-Hikari#1585] [Ideas]
Support different tax rates for different permission groups.
】This allows for the creation of a progressive tax law.

**Comment by CHERYLKURTENBACH** [comment]
[2023 Completed TAX ORGANIZER.zip](https://github.com/user-attachments/files/15522168/2023.Completed.TAX.ORGANIZER.zip)

**Reply by Ghost-chu** [reply]
[Author Reply] DO NOT DOWNLOAD THIS ATTACHMENT, It's a malware, I've reported it to Github Support.

**Comment by Ghost-chu** [comment]
Locked discussion cause it used for spread malware until the support removed the harmful comment.
